extensions/guacamole-auth-cas/src/licenses/LICENSE

```diff
@@ -0,0 +1,294 @@
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+==============================================================================
+
+APACHE GUACAMOLE SUBCOMPONENTS
+
+Apache Guacamole includes a number of subcomponents with separate copyright
+notices and license terms. Your use of these subcomponents is subject to the
+terms and conditions of the following licenses.
+
+
+AOP Alliance (http://aopalliance.sourceforge.net/)
+--------------------------------------------------
+
+    Version: 1.0
+    From: 'AOP Alliance' (http://aopalliance.sourceforge.net/members.html)
+    License(s):
+        Public Domain (bundled/aopalliance-1.0/LICENSE)
+
+Commons Codec (http://commons.apache.org/codec/)
+------------------------------------------------
+
+    Version: 1.4
+    From: 'Apache Software Foundation' (http://www.apache.org)
+    License(s):
+        Apache v2.0 (bundled/commons-codec/LICENSE-2.0.txt)
+
+
+Google Guice (https://github.com/google/guice)
+----------------------------------------------
+
+    Version: 3.0
+    From: 'Google Inc.' (http://www.google.com/)
+    License(s):
+        Apache v2.0 (bundled/guice-3.0/COPYING)
+
+Javax Servlet API (http://servlet-spec.java.net/)
+-------------------------------------------------
+
+    Version: 3.0.1
+    From: 'GlassFish Community' (http://servlet-spec.java.net/)
+    License(s):
+        CDDL 2 + GPLv2 with ClassPath Exception (bundled/javax.servlet-api/LICENSE)
+
+
+JSR-330 / Dependency Injection for Java (http://code.google.com/p/atinject/)
+----------------------------------------------------------------------------
+
+    Version: 1
+    From: 'JSR-330 Expert Group' (https://jcp.org/en/jsr/detail?id=330)
+    License(s):
+        Apache v2.0 (bundled/javax.inject-1/LICENSE-2.0.txt)
+
+JUnit (http://junit.org)
+------------------------
+
+    Version: 4.8.2
+    From: 'JUnit' (http://junit.org)
+    License(s):
+        CPL v1.0 (bundled/junit/LICENSE-1.0.txt)
+
+LOG4J (http://logging.apache.org/log4j/1.2/)
+--------------------------------------------
+
+    Version: 1.2.15
+    From: 'Apache Software Foundation' (https://www.apache.org)
+    License(s):
+        Apache v2.0 (bundled/log4j/LICENSE-2.0.txt)
+
+slf4j (http://www.slf4j.org)
```

Comment:
If this does end up bundled, we should exclude it. It will be provided at runtime by the Guacamole webapp, thus there is no need for the extension to include it, too:

https://github.com/apache/incubator-guacamole-client/blob/e8f4cd55155fef8e256b81b8047799eedcc009bd/guacamole/pom.xml#L266-L270